OpenEdXMobile/src/main/java/org/edx/mobile/social/google/GoogleProvider.java

```diff
@@ -42,55 +41,27 @@ public void getUser(Callback<SocialMember> callback) {
     public void getUserInfo(Context context,
                             SocialFactory.SOCIAL_SOURCE_TYPE socialType, String accessToken,
                             final SocialLoginDelegate.SocialUserInfoCallback userInfoCallback) {
-        // TODO: See if this should be implemented in Retrofit
-        //new GoogleUserInfoTask(context, userInfoCallback, accessToken).execute();
-
+        final String url = "https://www.googleapis.com/oauth2/v1/userinfo?access_token=" + accessToken;
```

Comment:
In my opinion we should only have our own APIs in the ApiConstants file, what I can do here is make this url a class level constant though.